applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_predictor.py

```diff
@@ -66,3 +67,32 @@ def _GetDefaultParameters(cls):
             "data_name"  : "UNSPECIFIED",
             "echo_level" : 0
         }""")
+
+
+class HistoricalDataAccessor:
```

Comment:
Is this class used else where ? 
If not then I suggest you put this class inside the predictor class. 
If it is used or you see that it can be used else where, put it in a separate file.

Reply:
I use it also in the derived classes

not sure what is better, I used the same as for another helper class:
https://github.com/KratosMultiphysics/Kratos/blob/663a16c1335a1f7967b29beb6fc3aa3976e9712a/applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_coupled_solver.py#L383-L386

I would either leave it as is or move both of those helpers to the respective classes
what do you think?

Reply:
I would move them helpers !!